nknd.go

```diff
@@ -150,14 +155,19 @@ func nknMain(c *cli.Context) error {
 		config.Parameters.Hostname = extIP
 	}
 
-	// Get local account
-	wallet, err := vault.GetWallet()
-	if err != nil {
-		return err
-	}
-	account, err := wallet.GetDefaultAccount()
-	if err != nil {
-		return errors.New("load local account error")
+	var wallet vault.Wallet
+	var account *vault.Account
+	for wallet == nil {
+		time.Sleep(time.Second * 3)
+		wallet, err = vault.GetWallet()
+		if err != nil {
+			fmt.Println(err)
+			continue
```

Comment:
We should continue trying to get the wallet only when onlyui is true. For other cases we should use the previous logic: return error if get wallet or account error